### ISSUE TITLE
[$10][MSFT-41] [Registration] - Missing Form Label.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2763]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Inspect Element
3. Observe Form Level

**Expected Result**
Registration form should have a form label.

**Actual Result**
Form label is not present.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: 
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=ae14bf9a%2D590b%2D4bb4%2Da40a%2D4c6822c880a4&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FCreeya%2FBug%201

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2763 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2763 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2763 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2763 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by nahidshahin** [comment]
PR https://github.com/appirio-tech/accounts-app/pull/249

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096408 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096408 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096408 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097535 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097535 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by mishacucicea** [comment]
@drasticdpk / @tosha5252 Is this issue referring to individual input fields not having a label?

**Comment by nithyaasworld** [comment]
This issue is only about Registration Form and not individual input fields.

@drasticdpk @tosha5252 Please "Pass" this if it meets the above. Thanks.

**Comment by dmessing** [comment]
Looks like this is done

**Comment by ajefts** [comment]
Verified in prod:

![image](https://user-images.githubusercontent.com/3812270/62154207-457ad700-b2d4-11e9-8bd0-67d4a83f7ab0.png)

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097535<br/><br/>```This is an automated message for crazyk via Topcoder X```
